test/evergreen.yml

```diff
@@ -2639,6 +2696,34 @@ tasks:
           #run for 2 hours ( 2 * 60 = 120 minutes), use default config
           format_test_script_args: -e "SEGFAULT_SIGNALS=all" -b "catchsegv ./t" -t 120
 
+  # FIXME-WT-8981 - Ubuntu PPC tests have been replaced by RHEL8 tests as of WT-8688, but the RHEL8 tests are currently disabled
+  # due to platform issues. On completion of WT-8981 this can be deleted.
+  - name: format-stress-sanitizer-ppc-test
+    tags: ["stress-test-ppc-ubuntu"]
+    # Set 2.5 hours timeout (60 * 60 * 2.5)
+    exec_timeout_secs: 9000
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          # FIXME-WT-8482: CC is set to the system default "clang" binary here as a workaround.
```

Comment:
this ticket is closed, is there some bugfix we're missing here?

Reply:
FIXME section removed in 6ebcabca06689c5485eabb04a16165e5693e7ecc